Makefile

```diff
@@ -233,6 +233,12 @@ check:
 	@echo "goimports"
 	@! goimports -l . | grep -vF 'No Exceptions'
 
+.PHONY: unused
+unused:
+# The -fields option generates false-positives for embedded mutex and
+# related types. See https://github.com/dominikh/go-unused/issues/18.
```

Comment:
Fyi, that issue is closed now (and `-fields` is on by default). Hopefully there won't be any noteworthy false positives anymore.